docs/video/smoothie/recipe.md

```diff
@@ -5,30 +5,42 @@ icon: simple/googlesearchconsole
 
 # Smoothie recipe
 
-The recipe (config file) is Smoothie's hardest part of it's learning curve, reading this and messing around with short clips is the best way to familiarize yourself with it. 
+The recipe (config file) is the hardest part of Smoothie's learning curve, reading this and messing around with short clips is the best way to familiarize yourself with it. 
 
+<!--
 The values which can be turned on or off (booleans) have a bunch of aliases for convenience, I advise typing y/n or 1/0 for short
 
-If there are some feature you're not interested in using, feel free to remove them from the file, it won't break a thing and act like you disabled it(1)
+I personally think this can go unmentioned, if you disagree and want to mention it, I'd recommend removing the term "booleans" (might be confusing for noobs)
```

Comment:
yeah revert them that's why i specify they're "values which can be turned on or off", there could also be a hyperlink to the wikipedia for boolean